src/import-export/azext_import_export/generated/action.py

```diff
@@ -0,0 +1,211 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=protected-access
+
+import argparse
+from knack.util import CLIError
+
+
+class AddReturnAddress(argparse.Action):
+    def __call__(self, parser, namespace, values, option_string=None):
+        action = self.get_action(values, option_string)
+        namespace.return_address = action
+
+    def get_action(self, values, option_string):  # pylint: disable=no-self-use
+        try:
+            properties = dict(x.split('=', 1) for x in values)
+        except ValueError:
+            raise CLIError('usage error: {} [KEY=VALUE ...]'.format(option_string))
+        d = {}
+        for k in properties:
+            kl = k.lower()
+            v = properties[k]
+            if kl == 'recipient-name':
+                d['recipient_name'] = v
+            elif kl == 'street-address1':
+                d['street_address1'] = v
+            elif kl == 'street-address2':
+                d['street_address2'] = v
+            elif kl == 'city':
+                d['city'] = v
+            elif kl == 'state-or-province':
+                d['state_or_province'] = v
+            elif kl == 'postal-code':
+                d['postal_code'] = v
+            elif kl == 'country-or-region':
+                d['country_or_region'] = v
+            elif kl == 'phone':
+                d['phone'] = v
+            elif kl == 'email':
+                d['email'] = v
```

Comment:
```suggestion
if kl in ['recipient-name', 'street-address1', ..., 'email']:
    d[kl] = v
```

Reply:
And the variable naming is hard to human to read. d, k, kl, v

Reply:
Actually the parameter name and the map key are not the same. For example, `state-or-province` is parameter name, `state_or_province` is the key, they are not the same. I prefer the current way because it is explicit.

Reply:
Oh. Sorry. I haven't noticed it.